cdap-app-templates/cdap-etl/cdap-etl-proto/src/main/java/io/cdap/cdap/etl/proto/v2/ETLBatchConfig.java

```diff
@@ -43,7 +44,7 @@
   private final List<ETLStage> postActions;
   // for backwards compatibility
   private final List<ETLStage> actions;
-  private final boolean service;
+  private final Boolean service;
```

Comment:
Why need to change?

Reply:
This is due to compatibility with etl config sent from UI. For example, if original ETLConfig does not set service field (service=null), resulting json after upgrade also should not have that service field. But if it was primitive boolean, it sets it as false. Hence changed it to use wrapper class.

Reply:
But isn't that we want the default value to be `false` anyway?

Reply:
Synced up offline. This is needed to minimize the changes between original ETLConfig and upgraded config to not confuse users.